server/eventGenerator/main.go

```diff
@@ -0,0 +1,27 @@
+package main
```

Comment:
Please move `eventGenerator` dir to the project's root and call it `generator`. There should already be something, so try refactor if possible to run all generations from your code